### PR TITLE
Add -B flag to FreeBSD pkgng.check() to regenerate the library dependency metadata

### PR DESCRIPTION
**Pull request by krionbsd**
### What does this PR do?
Add -B flag to pkgng.check() to regenerate the library dependency metadata for a package by extracting library requirement information from the binary ELF files in the package.

* Fix some typos in docstrings in pkgng.check() method.
* Generate additional tests for pkgng.check() method.

### CODE REVIEW

File: tests/unit/modules/test_pkgng.py [hunk 1]
```diff
@@ -539,3 +531,55 @@ def test_install_with_orphan(self):
...
+        pkg_cmd = MagicMock(return_value="")
+
+        with patch.dict(pkgng.__salt__, {"cmd.run": pkg_cmd}):
+            result = pkgng.check(recompute=True)
+            self.assertEqual(result, "")
+            pkg_cmd.assert_called_with(
+                ["pkg", "check", "-r"], output_loglevel="trace", python_shell=False,
+            )
+
+    def test_check_libs(self):
```
Comment (s0undt3ch) [inline tests/unit/modules/test_pkgng.py:574]:
```suggestion
    def test_checklibs(self):
```
To better associate with the parameter name?

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: tests/unit/modules/test_pkgng.py (hunk 1 above)
Comment (krionbsd) [reply tests/unit/modules/test_pkgng.py:574]: Native parameter is named `--shlibs`, but it doesn't sound too self-explanatory imho. For test case I put two underscores to separate method name and testcase, like `test_{method}_{flag}`, maybe `test_check_checklibs` makes more sense?

Comment (s0undt3ch) [reply tests/unit/modules/test_pkgng.py:574]: in that case, yes, `test_check_checklibs` makes more sense.

Comment (krionbsd) [reply tests/unit/modules/test_pkgng.py]: Fixed

**Review by s0undt3ch — APPROVED** [review]

**Review by Ch3LL — APPROVED** [review]

**Pull request merged**